File: app/src/main/java/br/com/alura/ceep/ui/coffemachine/presentation/view/Coffes.java
package br.com.alura.ceep.ui.coffemachine.presentation.view;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "coffes_table")
public class Coffes {

    @PrimaryKey
    public int uid;

    @ColumnInfo(name = "coffe_name")
    public String coffeName;

    @ColumnInfo(name = "quantity_capsules")
    public int quantityCapsules;

    public String getCoffeName () {
        return coffeName;
    }

    public int getQuantityCapsules () {
        return quantityCapsules;
    }

}

File: app/src/main/java/br/com/alura/ceep/ui/coffemachine/presentation/view/Coffes.kt
package br.com.alura.ceep.ui.coffemachine.presentation.view

import androidx.room.Entity
import androidx.room.PrimaryKey

//@Entity(tableName = "coffes_table")
@Entity
class Coffes (
    @PrimaryKey(autoGenerate = true) var id: Long,
    var name: String,
    var capsules: Long,
    var decription: String,
    var intensity: String,
    var quantity: String,
    var image: Int
)
//    @PrimaryKey var uid = 0
//    @ColumnInfo(name = "coffe_name")
//    var coffeName: String? = null
//
//    @ColumnInfo(name = "quantity_capsules")
//    var quantityCapsules = 0
